### PR TITLE
Arcadia sync

### PR DESCRIPTION
**Pull request by proller**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

### CODE REVIEW

File: dbms/programs/client/Client.cpp [hunk 1]
```diff
@@ -61,7 +61,7 @@
 #include <AggregateFunctions/registerAggregateFunctions.h>
 
 #if USE_READLINE
-#include "Suggest.h"
+#include "Suggest.h" // Y_IGNORE
```
Comment (alexey-milovidov) [inline dbms/programs/client/Client.cpp]: ?

Comment (proller) [reply dbms/programs/client/Client.cpp]: arcadia build without USE_READLINE and Suggest.h not copied to arcadia.
And on ya make run file "Suggest.h" is missing

Comment (alexey-milovidov) [reply dbms/programs/client/Client.cpp]: All files should be present in Arcadia.

Comment (proller) [reply dbms/programs/client/Client.cpp]: Will be soon. Now only needed for compilation copied.

**Pull request merged**